charts/argoconfig/values.yaml

```diff
@@ -28,38 +40,28 @@ appProject:
   sourceRepos:
     - '*'
 
-application:
-  # application.enabled -- Enable creating an Application resource
-  enabled: true
-  # application.namespace -- Namespace for Application resource
+apps:
+  # apps[0].enabled -- Enable creating an Application resource
+- enabled: true
+  # apps[0].name -- Name of app, used as suffix to app specific resources
+  name: "app"
+  # apps[0].namespace -- Namespace for Application resource
   namespace: argocd
-  # application.projectOverride -- Name of AppProject to install into
+  # apps[0].projectOverride -- Name of AppProject to install into
   # @default -- Fullname
   projectOverride: ~
   source:
-    # application.source.repoURL -- URL of sourece repo
+    # apps[0].source.repoURL -- URL of sourece repo
     repoURL: ""
-    # application.source.chart -- Name of chart in source repo
+    # apps[0].source.chart -- Name of chart in source repo
```

Comment:
I would be explicit and say `Helm chart`.

Reply:
I changed this to the exact string in the Argo CD CRD

Reply:
And refactored to link upstream anyway.